projects/packages/my-jetpack/src/class-initializer.php

```diff
@@ -663,7 +663,7 @@ public static function has_file_system_write_access() {
 			$write_access = 'yes';
```

Comment:
Doc block for this function could also use updating, where it says

> Returns true if the site has file write access to the plugins folder, false otherwise.

Reply:
Yeah, seems like a bugfix to me. :+1: